libraries/AP_Mount/AP_Mount_Siyi.h

```diff
@@ -0,0 +1,192 @@
+/*
+  Siyi gimbal driver using custom serial protocol
+
+  Packet format (courtesy of Siyi's SDK document)
```

Comment:
Can we include a link to that document in here?

Reply:
Not yet it seems.  It really should be here but it's not http://en.siyi.biz/en/Gimbal%20Camera/ZR10/download/ 

I'll ask Siyi if they can publish it publicly somewhere.  So far they've just provided links in the AP forums to a google drive